### PR TITLE
Turn stream option default on

### PR DESCRIPTION
**Pull request by graetzer**
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr-linux/4077/

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -1,6 +1,10 @@
 devel
 -----
 
+* AQL queries started from V8 in arangod (i.e. from a Foxx app) will not by
```
Comment (danielhlarkin) [inline CHANGELOG]: not -> now

**Comment by graetzer** [comment]
don't merge for now

**Comment by graetzer** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr-linux/4190/

**Comment by graetzer** [comment]
https://github.com/arangodb/arangodb/pull/8104

**Pull request closed without merging**